src/components/AdvancedSearchFilterInput.tsx

```diff
@@ -64,6 +72,17 @@ export default function AdvancedSearchFilterInput({
   };
 
   const selectedField = fields.find((field) => field.value === filterKey);
+  const selectedFieldOperators =
+    selectedField.operators !== undefined
+      ? operators.filter((element) => {
+          return selectedField.operators.includes(element.value);
+        })
+      : operators;
```

Comment:
Minor: This is fine and will work; but we could simplify it. Since there's only one statement in the filter function, we can drop the braces and the `return`. And because there's only one parameter, we can drop the parens. If we don't mind including `null` with the already-present `undefined`, we can also drop the comparison.
```
  const selectedFieldOperators = selectedField.operators
    ? operators.filter(op => selectedField.operators.includes(op.value))
    : operators;
```

Reply:
That it much more readable, thanks! `Eslint` doesn't like dropping the braces around `op`, so I added those back in, but otherwise I made this change.